src/osp/universe/coordinates.h

```diff
@@ -301,7 +302,7 @@ inline CoordTransformer coord_child_to_parent(
     return {
         .m_rotIn    = child.m_rotation,
         .m_c        = child.m_position,
-        .m_n        = -precisionDiff,
+        .m_n        = std::negate{}(precisionDiff),
```

Comment:
Honestly, I find this incredibly noisy and slightly more difficult to read. I may even argue that it's easier to miss when mixed with other variable names and functions. Perhaps just use brackets or something? `foo = ( - bar)`